File: src/main/java/de/nordgedanken/auto_hotkey/psi/AHKElementType.java
package de.nordgedanken.auto_hotkey.psi;

import com.intellij.psi.tree.IElementType;
import de.nordgedanken.auto_hotkey.AHKLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class AHKElementType  extends IElementType {
    public AHKElementType( @NotNull @NonNls String debugName) {
        super(debugName, AHKLanguage.INSTANCE);
    }
}

File: src/main/java/de/nordgedanken/auto_hotkey/lang/psi/AhkElementType.kt
package de.nordgedanken.auto_hotkey.lang.psi

import com.intellij.psi.tree.IElementType
import de.nordgedanken.auto_hotkey.lang.core.AhkLanguage
import org.jetbrains.annotations.NonNls

class AhkElementType(@NonNls debugName: String) : IElementType(debugName, AhkLanguage)
